changelog/unreleased/kong/move-sockets-to-subdir.yml

```diff
@@ -0,0 +1,3 @@
+message: Moved internal unix sockets to a subdirectory of the Kong prefix.
+type: feature
```

Comment:
@RobSerafini do you have any opinion on how we label this change? Seems closer to `feature` than `bugfix` for me, but I'll switch it to `bugfix` if the former is going to create drama for backporting.

Reply:
imho this is a bugfix. Since we're not adding any functionality. The future long-term fix, that would be a feature since it moves all runtime data.